nbi/buildspec.yml

```diff
@@ -17,36 +17,18 @@ phases:
       - ./$LCC_CONDA_INSTALLER -b -p $LCC_CONDA_INSTALL_DIR
   pre_build:
     commands:
-      - echo "Setting up conda"
```

Comment:
Why remove this?

Reply:
Accidentally modified it, adding back the change.